.github/scripts/notifyAboutIconRequest.js

```diff
@@ -0,0 +1,32 @@
+// @ts-nocheck
+// If the "calcite-ui-icons" label is added to an issue, generates a notification to the Icons team leads to review
+// The secret is formatted like so: icon-team-member-1, icon-team-member-2, icon-team-member-3
+// Note the script automatically adds the "@" character in to notify the icon team lead(s)
+
+/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
+module.exports = async ({ github, context }) => {
+  const { repo, owner } = context.repo;
+
+  const payload = /** @type {import('@octokit/webhooks-types').IssuesLabeledEvent} */ (context.payload);
+  const {
+    issue: { number },
+  } = payload;
+
+  const { ICONLEADS } = process.env;
```

Comment:
This doesn't match the `env` variable name defined [here](https://github.com/Esri/calcite-design-system/pull/11263/files#diff-3352a53ad58df5f5774810546c866b333c981fa2e7eaff4981e4e116b677b483R58).

Reply:
Updated!